app/_locales/en/messages.json

```
@@ -1,4 +1,10 @@
 {
+  "migrateSai": {
+    "message": "Dai is now \"Sai\" and Maker would like you upgrade your Sai tokens"
```

I see what you mean about the old wording, but this could be a bit misleading. It seems to imply Dai doesn't exist anymore.

```suggestion
    "message": "Your Dai is now \"Sai\" and Maker would like you upgrade your Sai tokens"
```
Maybe something like this would be more clear? I'm not totally sure it's an improvement... something along those lines anyway.

Also, is there something we could link for people to learn more about this migration? e.g. a "Learn more" link. I don't like having a notification that can't be dismissed, and then leaving no option but for users to "migrate", which they might be hesitant to click because of not understanding what it means.

---

I've updated this with new copy from @bdresser

---

What do you think about the "Learn More" suggestion? It looks like mobile added something like that as well.

---

I'll add it as the ignore text